examples/hydration-smiles/freesolv.yaml

```diff
@@ -0,0 +1,49 @@
+---
+options:
+  minimize: yes
+  verbose: yes
+  output_dir: .
+  number_of_iterations: 5000
+  temperature: 300*kelvin
+  pressure: 1*atmosphere
+  softcore_beta: 0.0
+
+molecules:
+  molecule:
+    filepath: freesolv-mini.smiles
+    openeye:
+      quacpac: am1-bcc
+    antechamber:
+      charge_method: null
+    select: all
+
+solvents:
+  RF:
+    nonbonded_method: PME
+    nonbonded_cutoff: 9*angstroms
+    clearance: 9*angstroms
+  vacuum:
+    nonbonded_method: NoCutoff
+
+systems:
+  hydration-system:
+    solute: molecule
+    solvent1: RF
```

Comment:
Do you want to call this solvent `water`?
Also, we probably want to change this to PME if we want to match `FreeSolv`.